Dockerfile

```diff
@@ -0,0 +1,21 @@
+#This is a docker file to generate a SQL server. It helps you to execute the integration tests and for development
```

Comment:
Let's do this in a separate PR so we can quickly review it there

Reply:
Sure, that makes perfect sense